manifests/0000_50_cluster-monitoring-operator_04-config.yaml

```diff
@@ -577,6 +577,10 @@ data:
     # Number of jaeger instances used certain agent strategy
     - '{__name__="jaeger_operator_instances_agent_strategies"}'
     #
+    # owners: (https://github.com/redhat-developer/application-services-metering-operator)
+    #
+    # The current amount of CPU used by Application Services products, labelled by product pillar.
+    - '{__name__="appsvcs_cpu_usage_cores"}'
```

Comment:
To prevent any cardinality explosion on the telemetry side if a new label is added to this metric, would you mind adding a new recording rule with the following expression:
```
sum by (prod_name) (appsvcs_cpu_usage_cores)
```

Reply:
No problem.

I'm new to this, so I'm not sure where the recording rule would go?

Reply:
I should go in https://github.com/openshift/cluster-monitoring-operator/blob/master/assets/cluster-monitoring-operator/prometheus-rule.yaml

Reply:
Should the metric in the config file be updated to then use the calculated metric based on the rule?

Reply:
The recording rule could also be shipped by your operator which makes it easier to maintain IMHO.

Reply:
I'm open to whatever approach is preferred. Having not done anything like this before I'm still somewhat finding my way.

If the rule was part of the operator, what is needed to achieve that?

If the rule is in the operator or in here, should the config be modified to use the rule as the metric and not the metric directly?

Reply:
Feedback on how to progress would be appreciated

Reply:
> If the rule was part of the operator, what is needed to achieve that?

You would need to create a PrometheusRule object containing the recoding rule you want to define. As we are doing in: https://github.com/openshift/cluster-monitoring-operator/blob/master/assets/cluster-monitoring-operator/prometheus-rule.yaml

 > If the rule is in the operator or in here, should the config be modified to use the rule as the metric and not the metric directly?

Yes exactly.

Reply:
Thanks @dgrisonnet

I will look to make the changes in the operator and update the PR when done

Reply:
I've pushed the change to use a metric recorded from a prometheus rule added by the operator